packages/react-components/react-combobox/src/components/Combobox/Combobox.test.tsx

```diff
@@ -7,16 +7,16 @@ describe('Combobox', () => {
   isConformant({
     Component: Combobox,
     displayName: 'Combobox',
-    primarySlot: 'trigger',
-    // don't test deprecated className export on new components
+    primarySlot: 'input',
+    // don't test derpecated className export on new components
```

Comment:
I like the new word derpecated 😂, but I think it's a tyop here?

Reply:
well now I like "derpecated" too 🤣